.github/workflows/build.yml

```
@@ -48,45 +53,80 @@ jobs:
       run: python setup.py sdist bdist_wheel
     - name: Upload wheels as artifacts
       uses: actions/upload-artifact@v2
+      if: ${{ matrix.python-version == '3.10' }}
       with:
-        name: python-packages
+        name: python-packages-${{ runner.os }}
         path: |
           dist/*.whl
           dist/*.tar.*
     - name: Install wheel and docs' dependencies
       run: pip install dist/*.whl -r docs/requirements.txt
+      shell: bash
     - run: pytest -vv -s tests/
       name: Run Python tests
     - name: Build html docs
       working-directory: docs
       run: sphinx-build . _build/html -W --keep-going -T
+    - name: Build dirhtml docs
+      working-directory: docs
+      run: sphinx-build . _build/dirhtml -W --keep-going -T
```

need to actually use the dirhtml builder here. I just realized I'm inspecting duplicates of the default html builder.
```suggestion
      run: sphinx-build . _build/dirhtml -W --keep-going -T -b dirhtml
```